lib/ddr/contacts.rb

```diff
@@ -0,0 +1,27 @@
+module Ddr
+  module Contacts
+    extend ActiveSupport::Autoload
+
+    class << self
+      attr_accessor :contacts
+    end
+
+    def self.get(contact_slug)
+      if contact_slug
+        load_contacts unless contacts
+        contacts[contact_slug]
+      end
+    end
+
+    def self.load_contacts
+      self.contacts = OpenStruct.new
+      contacts_file = File.join(Rails.root, 'config', 'contacts.yml')
+      YAML.load_file(contacts_file).each do |key, value|
+        contacts[key] = OpenStruct.new(value.merge('slug' => key))
+      end
+    rescue SystemCallError
+      Rails.logger.error("Unable to load Contacts file: #{contacts_file}")
```

Comment:
Does this mean that a config file has to be present to run the software? If so, I would prefer that the contacts simply be empty.